.github/workflows/crowdin.yml

```diff
@@ -85,4 +84,3 @@ jobs:
           --head "$HEAD_BRANCH" \
           --title "$TITLE" \
           --body "$BODY" \
-          --reviewer "$REVIEWER"
```

Comment:
@samwyndham maybe without the quotes
--reviewer $REVIEWER

Reply:
I't a known issue with some workarounds but I won't bother with them but create a follow up ticket - see https://github.com/cli/cli/issues/7671